pkg/initialdata/initialdata_test.go

```diff
@@ -87,19 +87,21 @@ func (m *mockDatastore) Exists(ctx context.Context, collection string, id int) (
 	return ok, nil
 }
 
-func (m *mockDatastore) Create(ctx context.Context, fqid string, fields map[string]json.RawMessage) error {
-	ss := strings.Split(fqid, "/")
-	collection := ss[0]
-	id, _ := strconv.Atoi(ss[1])
+func (m *mockDatastore) Create(ctx context.Context, creatables map[string]map[string]json.RawMessage, migrationIndex int) error {
+	for fqid, fields := range creatables {
+		ss := strings.Split(fqid, "/")
+		collection := ss[0]
+		id, _ := strconv.Atoi(ss[1])
```

Comment:
specially in tests you should check the error. Testdata should contain invalid data. And your mock should handle invalid data correctly. The same in line 96

Reply:
Done.